petstore/spec/tspconfig.yaml

```diff
@@ -13,3 +14,5 @@ options:
     express: true
   "@typespec/http-client-java":
     emitter-output-dir: "{project-root}/../clients/java"
+  "@azure-tools/typespec-ts":
+    emitter-output-dir: "{project-root}/../clients/javascript"
```

Comment:
we need to ensure the code could be generated as this config. could you verify that?